bundles/org.openhab.binding.carnet/src/main/resources/OH-INF/config/config.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="bridge-type:carnet:account">
+		<parameter-group name="authentication">
+			<label>Authentication</label>
+			<description>Authentication settings.</description>
+		</parameter-group>
+
+		<parameter name="brand" type="text" required="true">
+			<label>Brand</label>
+			<description>CarNet Brand.</description>
+			<options>
+				<option value="Audi">Audi</option>
+			</options>
+			<default>Audi</default>
+		</parameter>
+		<parameter name="country" type="text" required="true">
+			<label>Market (Country)</label>
+			<description>Market in which is associated (must not match the country where the vehicle was bought).</description>
+			<options>
+				<option value="DE">Europe</option>
+				<option value="United States">US</option>
+				<option value="China">CN</option>
```

Comment:
Are the notations interchanged by intention?

Reply:
fixed